contrib/seekable_format/zstd_seekable.h

```diff
@@ -154,21 +155,33 @@ ZSTDLIB_API size_t ZSTD_seekable_writeSeekTable(ZSTD_frameLog* fl, ZSTD_outBuffe
 ZSTDLIB_API ZSTD_seekable* ZSTD_seekable_create(void);
 ZSTDLIB_API size_t ZSTD_seekable_free(ZSTD_seekable* zs);
 
+/*===== Independent seek table management =====*/
+ZSTDLIB_API size_t ZSTD_seekable_copySeekTable(ZSTD_seekable* zs, ZSTD_seekTable** out);
```

Comment:
minor : naming.

I expect `copy` to represent an operation which would copy a content into an already allocated buffer.
But here this is different. 
This operation effectively allocates the required memory buffer(s), and then forwards ownership to the caller.
There is also a risk of failure, that the caller must check.

Naming is hard.
It looks to me that this is actually a _creation_ operation,
which _also_ requires a specific input object to proceed.
In order to respect the established naming convention of the `zstd` project, I would recommend the following prototype :

`ZSTD_seekTable* ZSTD_seekTable_create_fromSeekable(const ZSTD_seekable* zs);`

Note that the change of signature is non trivial :
- `create` is pretty clear in signifying that it allocates an object, that the caller must then manage.
- like any allocation operation, it may fail, and the failure will be signaled by a `NULL` return value, which is a common `C` idiom
- the argument is `const`, meaning that its content is guaranteed to remain intact during the operation. This is a _very strong_ guarantee, which scales greatly in multi-threaded environments, such an opportunity should never be missed.
- the naming scheme leaves room for other forms of creators, such as a potential `_fromInput()` for example